src/mock/v2.0/intent-l.ts

```diff
@@ -0,0 +1,30 @@
+import { closeWindowOnCompletion, onFdc3Ready } from "./mock-functions";
```

Comment:
this shouldn't be intent-L, the intent-* mocks are for the Intents tests.  This is a basic test, so maybe call it "basic"?